sim/rogue/vanish.go

```diff
@@ -33,11 +33,11 @@ func (rogue *Rogue) registerVanishSpell() {
 			IgnoreHaste: true,
 			CD: core.Cooldown{
 				Timer:    rogue.NewTimer(),
-				Duration: time.Second * time.Duration(300-45*rogue.Talents.Elusiveness),
+				Duration: time.Second*time.Duration(300 - float64(45*rogue.Talents.Elusiveness)),
 			},
 		},
 		ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
-			if has4Pc {
+			if has4PcT1 {
```

Comment:
We could turn this into an attribute to avoid checking for the set bonus here but up to you on that! I don't feel strongly